drivers/syslog/vsyslog.c

```diff
@@ -166,66 +168,66 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
   switch (priority)
     {
       case LOG_EMERG:   /* Red, Bold, Blinking */
-        ret += lib_sprintf(&stream.public, "\e[31;1;5m");
+        offset += sprintf(offset, "\e[31;1;5m");
```

Comment:
can we use array to map priority to the color string?

Reply:
done